sagemaker-training-compiler/tensorflow/multiple_gpu_single_node/vision-transformer.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<p>Add the following:</p><p>Note that SageMaker collects metrics at a resolution of 1 minute and metrics for some epochs might be lost.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>